microprofile/health-smallrye/src/main/java/org/wildfly/extension/microprofile/health/deployment/CDIExtension.java

```diff
@@ -98,8 +96,23 @@ private void afterDeploymentValidation(@Observes final AfterDeploymentValidation
         }
     }
 
+    private BeanManager getBeanManager(Module module, BeanManager eventBeanManager) {
+        return tryGetBeanManagerFromJndiLookup()
```

Comment:
I don't think it is necessary in the DUP to use JNDI to look up the BeanManager. Using directly `CDI.current().getBeanManager()`is fine